src/traces/scattermapbox/attributes.js

```diff
@@ -87,6 +87,23 @@ module.exports = overrideAll({
                 'are only available for *circle* symbols.'
             ].join(' ')
         },
+        angle: {
+            valType: 'number',
+            dflt: null,
```

Comment:
Can we set `dflt: 0` and then instead of `if('angle' in trace.marker)`, `if(trace.marker.angle)`? It's more useful for anyone inspecting the `fulltrace` (including chart studio) to be explicit about the default.

Reply:
I think doing if(trac.marker.angle) means that entering now generates different code. I think it's better to keep it such that not providing an angle at all doesn't inject any additional mapbox code, and specifying one (even if it's 0), does. lmk if you think that is trumped by other concerns and I can switch to `dftl: 0`

Reply:
On our side providing an explicit default is strongly preferred, rather than omitting the default. There should be no difference for the user between not providing an angle and providing zero, and `fullTrace` should reflect that by being identical in the two cases. Aside from chart studio and helping users learn about available attributes, there are occasional practical benefits such as `Plotly.react` knowing it doesn’t need to do anything if you change from undefined to zero angle. 

I don’t really care whether we send the zero to mapbox or not, as I can’t really imagine the default ever changing from zero on their side.